app/uk/gov/hmrc/cgtpropertydisposals/controllers/upscan/UpscanController.scala

```diff
@@ -23,7 +23,7 @@ import play.api.mvc.{Action, AnyContent, ControllerComponents}
 import uk.gov.hmrc.cgtpropertydisposals.controllers.actions.AuthenticateActions
 import uk.gov.hmrc.cgtpropertydisposals.models.ids.DraftReturnId
 import uk.gov.hmrc.cgtpropertydisposals.models.upscan.UpscanCallBackEvent._
-import uk.gov.hmrc.cgtpropertydisposals.models.upscan.{UpscanCallBackEvent, UpscanFileDescriptor, UpscanInitiateReference, UpscanSnapshot}
+import uk.gov.hmrc.cgtpropertydisposals.models.upscan.{UpscanCallBackEvent, UpscanFileDescriptor, UpscanInitiateReference}
```

Comment:
This trio of case classes will get trashed after this PR. This is what I was referring to about cleaning up the whole area and building a nice interface and data model